dpdk-ip-pipeline-cli/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM python:2
+COPY . /app
```

Comment:
Can we add a `.dockerignore` including the Dockerfile itself ?
https://docs.docker.com/engine/reference/builder/#dockerignore-file